.github/workflows/cont_integration.yml

```
@@ -106,6 +106,34 @@ jobs:
       - name: Test
         run: cargo test --features test-${{ matrix.blockchain.name }} ${{ matrix.blockchain.name }}::bdk_blockchain_tests
 
+  test-blockchains-esplora:
+    name: Test esplora-${{ matrix.backend.name }}
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        backend:
+          - name: ureq
+          - name: reqwest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-${{ github.job }}-${{ hashFiles('**/Cargo.toml','**/Cargo.lock') }}
+      - name: Setup rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - name: Test
+        run: cargo test --no-default-features --features test-esplora,use-esplora-${{ matrix.backend.name }} esplora::bdk_blockchain_tests
```

As @RCasatta suggested, this will work:
```suggestion
        run: cargo test --no-default-features --features test-esplora,use-esplora-${{ matrix.backend.name }} esplora::${{ matrix.backend.name }} ::bdk_blockchain_tests
```